hail/python/test/hail/linalg/test_linalg.py

```diff
@@ -1028,6 +1028,32 @@ def assert_same_columns_up_to_sign(a, b):
         s = x.svd(compute_uv=False, complexity_bound=0)
         assert np.all(s >= 0)
 
+    def test_ndarray(self):
```

Comment:
I'd suggest we put this in `test_nd.py` with the rest of the other ndarray tests